src/classes/issues-processor.ts

```diff
@@ -810,7 +810,7 @@ export class IssuesProcessor {
 
     const filteredComments = comments.filter(
       comment =>
-        comment.user?.type === 'User' &&
+        comment.user?.type !== 'Bot' &&
```

Comment:
Are there supposed to be two equal signs, or just 1?

Reply:
with typescript i thought it was best to use the triple comparison to avoid any implicit equality comparison